CONTRIBUTING.md

```diff
@@ -166,8 +166,7 @@ Makefile targets. There targets rely on environment variables to point to the
 correct environment.
 
 Environment variables:
-- You **need** to set ESX and either VM_IP (in which case we'll use 1 VM) or
-both VM1 and VM2 environment variables
+- You **need** to set ESX and VM1, VM2 and VM3 environment variables
```

Comment:
and if I don't , will the test tell me that ? It *has* to IMO, it is super cheap to check for .At the very least, tell how the failure will look like if the vars are not set up